pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -146,15 +146,17 @@ svg {
     // when possible.
     // https://github.com/patternfly/patternfly/issues/4115
     &.ct-compact {
-        display: flex;
-        flex-wrap: wrap;
+        @media screen and (max-width: $pf-global--breakpoint--lg - 1) {
```

Comment:
That causes a compilation failure:
 Error: Undefined variable: "$pf-global--breakpoint--lg".
        on line 149 of pkg/lib/patternfly/patternfly-4-overrides.scss
>>         @media screen and (max-width: $pf-global--breakpoint--lg - 1) {

Reply:
It's a SASS variable from pkg/lib/_global-variables.scss. We have to use it.